content/python/concepts/operators/operators.md

```diff
@@ -32,47 +35,42 @@ Python divides the operators in the following groups:
 
 ## Arithmetic Operators
 
-Arithmetic operators are used with numeric values to perform common mathematical operations:
+Python has the following arithmetic operators:
 
-| Operator | Name           | Example  |
-| -------- | -------------- | -------- |
-| `+`      | Addition       | `a + b`  |
-| `-`      | Subtraction    | `a - b`  |
-| `*`      | Multiplication | `x * y`  |
-| `/`      | Division       | `x / y`  |
-| `%`      | Modulus        | `x % y`  |
-| `**`     | Exponentiation | `x ** y` |
-| `//`     | Floor division | `x // y` |
+- `+`: Addition, returns the sum of two numbers.
+- `-`: Subtraction, returns the difference of two numbers.
+- `*`: Multiplication, returns the product of two numbers.
+- `/`: Division, returns the quotient of two numbers.
+- `**`: Exponentiation, returns the value of one number raised to the power of another.
+- `%`: Modulus, returns the remainder of one number divided by another.
+- `//`: Floor division, returns integer quotient of two numbers.
 
 ## Assignment Operators
 
-Assignment operators are used to assign values to variables:
+Python includes the following assignment operators:
 
-- `+=`
-- `-=`
-- `*=`
-- `/=`
-- `%=`
+- `=`: Assigns the value on the left to the variable on the right.
+- `+=`: Adds the value on the right to the variable on the left then assigns the result to the variable on the left.
+- `-=`: Subtracts the value on the right from the variable on the left then assigns the result to the variable on the left.
+- `*=`: Multiplies the value on the right with the variable on the left then assigns the result to the variable on the left.
+- `/=`: Divides the value on the right into the variable on the left then assigns the result to the variable on the left.
+- `%=`: Calculates the modulus of the variable on the right with the value on the left and assigns the result to the variable on the left.
```

Comment:
```suggestion
- The `=` operator assigns the value on the left to the variable on the right.
- The `+=` operator updates a variable by incrementing its value and reassigning it.
- The `-=` operator updates a variable by decrementing its value and reassigning it.
- The `*=` operator updates a variable by multiplying its value and reassigning it.
- The `/=` operator updates a variable by dividing its value and reassigning it.
- The `%=` operator updates a variable by calculating its modulus against another value and reassigning.
```

Reply:
Hmm, I guess this can read more fluidly for some. I'm not opposed to making these (and the above list) full sentences.

Reply:
Totes fine with either. Your call. 🙂